ReSharper.FSharp/src/FSharp.Psi.Intentions/src/QuickFixes/UpdateLiteralConstantFix.fs

```diff
@@ -0,0 +1,70 @@
+namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+
+open FSharp.Compiler.Symbols
+open JetBrains.ReSharper.Plugins.FSharp.Psi
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.Highlightings
+open JetBrains.ReSharper.Psi.ExtensionsAPI
+open JetBrains.ReSharper.Psi.Tree
+open JetBrains.ReSharper.Resources.Shell
+
+type UpdateLiteralConstantFix(error: ValueNotContainedMutabilityLiteralConstantValuesDifferError) =
+    inherit FSharpQuickFixBase()
+    
+    let tryFindSigFile (topRefPat: ITopReferencePat) =
+        let containing = topRefPat.GetContainingTypeDeclaration()
+        let decls = containing.DeclaredElement.GetDeclarations()
+        decls |> Seq.tryFind (fun d -> d.GetSourceFile().IsFSharpSignatureFile)
+        
+    let tryFindSigBindingSignature sigMembers =
+        sigMembers
+        |>  Seq.tryPick(fun m ->
+                let bindingSignature = m.As<IBindingSignature>()
+                match bindingSignature with
+                | null -> None
+                | _ ->
+                    match error.Pat.Binding.HeadPattern with
+                    | :? IReferencePat as implPat ->
+                        match bindingSignature.HeadPattern with
+                        | :? IReferencePat as sigRefPat when
+                            implPat.DeclaredName = sigRefPat.DeclaredName
+                            -> Some bindingSignature
```

Comment:
Please put the arrow on the previous line.

Reply:
Done